Counter.tsx

```diff
@@ -0,0 +1,41 @@
+import React, {Component} from 'react';
+import {Button, Text, View} from 'react-native';
+import styles from './styles';
+
+interface Props {}
+interface State {
+  count: number;
+}
+
+export default class Counter extends Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = { count: 0 };
+  }
+
+  changeCount(changeAmount: number) {
```

Comment:
Change this to an arrow function. `changeCount = (changeAmount: number) =>` This way you get bind for free because arrow functions do not have their own 'this'.